channeldb/graph.go

```diff
@@ -1800,6 +1850,9 @@ func delChannelEdge(edges, edgeIndex, chanIndex, zombieIndex,
 		}
 	}
 
+	updateEdgePolicyDisabledIndex(edges, cid, false, false)
```

Comment:
some comments would help explain what's happening here, e.g. we remove the entries from the disabled index since we are deleting the channel